parm/config/gefs/config.resources

```diff
@@ -272,7 +272,7 @@ case ${step} in
     export walltime_gefs="00:30:00"
     export ntasks_gefs=1
     export threads_per_task_gefs=1
-    export tasks_per_node_gefs="${ntasks}"
+    export tasks_per_node_gefs="${ntasks_gefs}"
```

Comment:
@DavidHuber-NOAA Adjustments needed to be made to the extractvars resources due to changes made from PR #2727. Could you let me know that this is correct?  I changed `ntasks` to `ntasks_gefs` because when `ntasks` was used here, I got the following error when setting up my experiment: 
```
  File "/lfs/h2/emc/ens/noscrub/eric.sinsky/GEFS/offsetfixes/workflow/rocoto/tasks.py", line 186, in get_resource
    nodes = int(np.ceil(float(ntasks) / float(ppn)))
ValueError: could not convert string to float: 
```

Reply:
@EricSinsky-NOAA Did you update your submodules locally (i.e. `git submodule update --init --recursive`)?  If this does not resolve the issue, can you point me to your experiment directory?

Reply:
I did update my submodules before setting up my experiment. My experiment can be set up successfully when I use `ntasks_gefs` instead of `ntasks`. Here is my experiment directory: `/lfs/h2/emc/ens/noscrub/eric.sinsky/GEFS/expdir/customexp/offsetfixes`.

Reply:
@EricSinsky-NOAA I apologize, I misread your initial post.  Yes, this looks correct.  Thanks for catching my bug.